### PR TITLE
gst-validate 1.4.0 (new formula)

### PR DESCRIPTION
**Pull request by chul0812**
gst-validate is a suite of tools to run integration tests for gstreamer

### CODE REVIEW

File: Library/Formula/gst-validate.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
+require 'formula'
```
Comment (bfontaine) [inline Library/Formula/gst-validate.rb]: You can remove this line, it’s not necessary anymore.

File: Library/Formula/gst-validate.rb [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+class GstValidate < Formula
+  homepage "http://gstreamer.freedesktop.org/data/doc/gstreamer/head/gst-validate/html/"
+  url "http://gstreamer.freedesktop.org/src/gst-validate/gst-validate-1.4.0.tar.xz"
+  version "1.4.0"
```
Comment (bfontaine) [inline Library/Formula/gst-validate.rb]: You can remove the version here, Homebrew already gets it from the URL.

**Comment by chul0812** [comment]
Sorry, I am not familiar with github review system yet.
I just intent to update first commit but the second commit is created with improper commit-message.
Is it better to create new pull-request instead of make it several commit?

**Comment by bfontaine** [comment]
No, it’s fine to make several commits, they’ll be squashed into one commit by the Homebrew maintainer who’ll merge your PR.

If you’re familiar with `git`, you can squash them yourself with `git rebase -i` then `push -f` on GitHub, but it’s ok if you don’t. I’m not an Homebrew maintainer, but I have some experience with their review process.

**Comment by chul0812** [comment]
That sound great, thank you for your reply bfontain.

**Comment by mikemcquaid** [comment]
A few warnings that need fixed: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/24071/version=yosemite/testReport/junit/brew-test-bot/yosemite/audit_gst_validate___strict/

**Comment by xu-cheng** [comment]
Thanks for the pull request! 🎉 Homebrew depends on contributions from community members like you and we're grateful for your support.

**Comment by tdsmith** [comment]
This is installing with `python2` in the shebang, which only worked because gobject-introspection introduced a bogus dependency on Homebrew python: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/24252/version=mavericks/testReport/junit/brew-test-bot/mavericks/test_gst_validate/

**Comment by tdsmith** [comment]
Opened #38669.

**Pull request closed without merging**